### ISSUE TITLE
Beginner JavaScript: Translate Better—or at least different—songs from English to French

### ISSUE THREAD

**Issue by PhilipHarney** [CoderDojo-Content/content-hack#271]
**If this is your first time working on *Beginner JavaScript* then you will need to be added as a collaborator. You can find instructions on how to do this at http://dojo.soy/gitbook-collab**
1. Make sure you have assigned this task card to yourself and moved it to the **In Progress** column.
2. Go to https://www.gitbook.com/book/coderdojo/beginner-javascript-sushi/edit and create a change request named **Translation of Better—or at least different—songs (better_different_songs.md) to French**. See how to do this at http://dojo.soy/gitbook-makecr
3. Switch the view to Français fr. See how to do this at http://dojo.soy/gitbook-changelang
4. Translate the English text into French. For an overview of what to translate, see http://dojo.soy/translation-guidelines.
5. Once the translation is complete, save your work by clicking the **Publish** button.
6. Move this task card into the **Review** column.
7. If you have time, pick another task from the **Next** column!
